docs/UserGuide.md

```diff
@@ -753,19 +762,28 @@ Exports contacts based on contacts and their details stored in KonTActs.
 
 <box type="warning" icon=":fa-solid-circle-exclamation:" light>
 
-* File path can be relative or absolute but must end with .csv
-  </box>
-* You must have write permission in order to export CSV file to specified location
+* File path can be relative or absolute but must end with ".csv". 
+  * If using relative path, you may use ".." to move up one directory level (you may refer to example below).
+  * Although ~ symbol is commonly used as home, it will not be the case when using export. Please use the full
+  home path instead.
+      </box>
+* You must have write permission in order to export CSV file to specified location.
 
 <box type="definition" icon=":fa-solid-book:" light>
 
 <md>**Examples:**</md>
 
 * `export path/data/group12.csv` will export the contacts in KonTActs as a csv file (group12.csv at the 
-relative path given)
-* Alternatively, `ex p/data/group12.csv` will do the exact same. It uses the shortcut command and parameter. 
+RELATIVE path given).
+* Alternatively, `ex p/data/group12.csv` will do the exact same. It uses the shortcut command and parameter.
+* `export path//data/group12.csv` will export the contacts in KonTActs as a csv file (group12.csv at the
+ABSOLUTE path given - absolute paths specify the exact location in the file system, starting from the root directory,
```

Comment:
Maybe break the long sentence into 2 for better reading. "... ABSOLUTE path given. Absolute paths specify..."